source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -665,8 +665,8 @@ The event API here is assumed to be like the standard `Python Event
             # was successful and included the topologyVersion field, or the
             # previous response included the moreToCome flag, or the server
             # has just transitioned to Unknown from a network error.
-            if (description.type != Unknown and description.topologyVersion != Null or
-                    connection.moreToCome or
+            if (description.type != Unknown and description.topologyVersion != Null) or
```

Comment:
`if (description.type` -> `if ((description.type` (missing parenthesis)

Reply:
New commit splits this logic over multiple lines so the extra parentheses aren't needed.